packages/lib/src/RegistrationWall.wc.svelte

```diff
@@ -1,171 +1,92 @@
 <svelte:options customElement="sesamy-registration-wall" />
 
 <script lang="ts">
+  import type { SesamyAPI } from "@sesamy/sesamy-js";
   import AppleLogo from "./assets/AppleLogo.svelte";
   import FacebookLogo from "./assets/FacebookLogo.svelte";
   import GoogleLogo from "./assets/GoogleLogo.svelte";
-
-  function handleClick() {
-    console.log("Button clicked");
-  }
+  import Base from "./Base.svelte";
+
+  const login = async (api: SesamyAPI) => {
+    try {
+      await api.auth.loginWithRedirect();
+    } catch (error) {
+      console.error("Login failed:", error);
+    }
+  };
 </script>
 
-<div class="registration-wall">
-  <slot name="title"
-    ><h1 class="text-3xl font-bold underline">
-      Skapa ett gratis konto eller logga in
-    </h1></slot
+<Base let:api>
+  <div
+    class="max-w-[800px] mx-auto p-5 text-center bg-[#f8f8f5] border border-[#e9e9da] rounded-lg font-sans"
```

Comment:
Colors should probably move into design tokens at some point. This works fine for now.